### ISSUE TITLE
[question] Is it possible to add Temporal Anti-Aliasing?

### ISSUE THREAD

**Issue by Casul51** [DadSchoorse/vkBasalt#90]
It would be killer feature for this project but I guess it's impossible, as I understand it should be inside game engine...

**Comment by Casul51** [comment]
(actually I've found this https://github.com/BlueSkyDefender/Depth3D reshade's TAA implementation probably)

**Comment by DadSchoorse** [comment]
Well, TAA isn't one technique, it's simply the principal of reusing old frames to get data for anti-aliasing. To make that work well you need to apply jittering to the viewport, with is not possible to do in a generic way, and ideally you also want motion vectors, which vkBasalt can't get either. BlueSkyDefender's TAA tries to work around that (imo it doesn't work well enough to be worth it). But even that need depth information, so you can't use it in vkBasalt.

TLDR: No, TAA isn't really possible due to missing data and jittering.
